### PR TITLE
Make Caps Lock delay more reasonable

### PR DESCRIPTION
**Pull request by drashna**
This reduces the default timeout back to what was the hardcoded default was, but keeps the customizational options.

### CODE REVIEW

**Comment by vomindoraan** [comment]
Lines 174 and 175 in `docs/config_options.md` should also be updated to reflect the change from 200 to 80.

**Comment by drashna** [comment]
Well, 80 was the old default, and it wasn't enough on some systems.  I was .... overzealous on fixing it.  

But glad to hear that.   And docs change incoming.  Nice catch.

**Comment by vomindoraan** [comment]
Just out of curiosity, if I set this value to 50 in my `config.h`, I shouldn't have any trouble on Windows and Linux, right?

File: docs/config_options.md [hunk 1]
```diff
@@ -171,8 +171,8 @@ If you define these options you will enable the associated feature, which may in
   * how long for the Combo keys to be detected. Defaults to `TAPPING_TERM` if not defined.
 * `#define TAP_CODE_DELAY 100`
   * Sets the delay between `register_code` and `unregister_code`, if you're having issues with it registering properly (common on VUSB boards). The value is in milliseconds.
-* `#define TAP_HOLD_CAPS_DELAY 200`
-  * Sets the delay for Tap Hold keys (`LT`, `MT`) when using `KC_CAPSLOCK` keycode, as this has some special handling on MacOS.  The value is in milliseconds, and defaults to 200ms if not defined. 
+* `#define TAP_HOLD_CAPS_DELAY 80`
+  * Sets the delay for Tap Hold keys (`LT`, `MT`) when using `KC_CAPSLOCK` keycode, as this has some special handling on MacOS.  The value is in milliseconds, and defaults to 80ms if not defined.  For MacOS, you may want to set this to 200+.
```
Comment (vomindoraan) [inline docs/config_options.md]: Tiny little change: units are usually separated from the quantity by a space. Also, remove the double space after the period.
```suggestion
  * Sets the delay for Tap Hold keys (`LT`, `MT`) when using `KC_CAPSLOCK` keycode, as this has some special handling on MacOS.  The value is in milliseconds, and defaults to 80 ms if not defined. For MacOS, you may want to set this to 200 or higher.
```

**Comment by drashna** [comment]
Shouldn't, no.  I can't be 100% certain without testing, though.

**Comment by fauxpark** [comment]
Caps Lock has no delay on Windows and Linux, it's only macOS that needs this increased.

**Comment by vomindoraan** [comment]
Alright, thanks guys.

File: docs/config_options.md [hunk 2]
```diff
@@ -171,8 +171,8 @@ If you define these options you will enable the associated feature, which may in
   * how long for the Combo keys to be detected. Defaults to `TAPPING_TERM` if not defined.
 * `#define TAP_CODE_DELAY 100`
   * Sets the delay between `register_code` and `unregister_code`, if you're having issues with it registering properly (common on VUSB boards). The value is in milliseconds.
-* `#define TAP_HOLD_CAPS_DELAY 200`
-  * Sets the delay for Tap Hold keys (`LT`, `MT`) when using `KC_CAPSLOCK` keycode, as this has some special handling on MacOS.  The value is in milliseconds, and defaults to 200ms if not defined. 
+* `#define TAP_HOLD_CAPS_DELAY 80`
+  * Sets the delay for Tap Hold keys (`LT`, `MT`) when using `KC_CAPSLOCK` keycode, as this has some special handling on MacOS.  The value is in milliseconds, and defaults to 80 ms if not defined. For MacOS, you may want to set this to 200 or higher.
```
Comment (vomindoraan) [inline docs/config_options.md]: Just noticed: macOS spelling 🙄
```suggestion
  * Sets the delay for Tap Hold keys (`LT`, `MT`) when using `KC_CAPSLOCK` keycode, as this has some special handling on MacOS.  The value is in milliseconds, and defaults to 80 ms if not defined. For macOS, you may want to set this to 200 or higher.
```

**Pull request merged**